util/config/exec.go

```diff
@@ -0,0 +1,20 @@
+package config
+
+import (
+	"os"
+	"time"
+
+	"github.com/argoproj/pkg/exec"
+)
+
+func timeout() time.Duration {
+	duration, err := time.ParseDuration(os.Getenv("ARGOCD_EXEC_TIMEOUT"))
```

Comment:
Environment variables never really change once the program starts, I think we can calculate a private `time.Duration` value as part of an init() function to optimize all of the unnecessary calls to `time.ParseDuration`, `os.Getenv` `timeout()`, which will only ever get set once.

Reply:
kk

Reply:
um, how do I test it?